packages/kokoas-client/src/pages/projInvoice/fieldComponents/selectInvoices/InvoicesDialogContent.tsx

```diff
@@ -0,0 +1,75 @@
+import { DialogContent, FormControlLabel, Radio, RadioGroup, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import { GetInvoicesByCustGroupId, useInvoicesByCustGroupId } from 'kokoas-client/src/hooksQuery';
+import WarningAmberIcon from '@mui/icons-material/WarningAmber';
+import { ListItemInvoices } from './ListItemInvoices';
+import { TInvoiceStatus } from '../../form';
+
+export interface InvoicesDialogContentProps {
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  custGroupId: string
+  value: string
+}
+
+export const InvoicesDialogContent = (props: InvoicesDialogContentProps) => {
+  const { onChange, custGroupId, value } = props;
+
+  const { data } = useInvoicesByCustGroupId<GetInvoicesByCustGroupId>(custGroupId);
```

Comment:
変更不要。
```suggestion
  const { data } = useInvoicesByCustGroupId(custGroupId);
```

デフォルトのuseInvoicesByCustGroupIdのジェネリックタイプから、既にタイプが推論されているため、ここで明示的に定義しなくてもいいと思います。

Reply:
承知しました。
別途、修正できるタイミングで対応します。